### ISSUE TITLE
Multisig verification and transaction signing

### ISSUE THREAD

**Issue by sasha-astiadi** [threefoldfoundation/tft#95]
# TFT-BSC v1.0.0

## **Background  / Description**
BSC V.10-Beta We already have a singlesig bridge service that runs on a single master key. we need to create multisig service and a fee wallet/

# Requirements

**- Multisig TFT-stellar to TFT-BSC transaction**

As a user, if you convert tft-stellar to tft-bsc convertable using bsc bridge, it costs 50 tft as transaction fee. Now we need to get the TFT out of the bridge wallet. The bridge wallet is distributed with multisig, so it needs to be co-signed in order to complete the transaction. Multisig creates more secure transaction, and that none can steal or compromise it.​​​​​​​

**- Multisig TFT-BSC to TFT-stellar transaction**

As a user, if you convert TFT-BSC to TFT-Stellar, it costs 1 tft as transaction fee. Now we need to get the withdrawn TFT  out of the bridge wallet to the user's stellar wallet (minus TFT as transaction fee. The bridge wallet is distributed with multisig, so it needs to be co-signed in order to complete the transaction. Multisig creates more secure transaction, and that none can steal or compromise it.​​​​​​​

## User Story

Create services to support multisig verification and transaction signing

### Tickets

- [x] transfer transaction fee from the bridge wallet to TF fee wallet #42 

- [x]  Revalidate the mutlisig transaction process of the entire project (tft -stellar- tft-bsc chain, tft-bsc-tft-stellar transaction) #94 

- [x]  improve the docker build and the helm chart #51

- [x] Set up a slave and deploy BSC bridge in Lochristi https://github.com/threefoldtech/itenv_tfgridv2/issues/28

- [ ]  optional / nice to have : deploy it on ha cluster https://github.com/threefoldtech/itenv_tfgridv2/issues/29

**Comment by sasha-astiadi** [comment]
blocked on https://github.com/threefoldtech/itenv_tfgridv2/issues/26

**Comment by robvanmieghem** [comment]
@sasha-astiadi 
I want to close this issue as the last item was optional/nice to have and it is more an operational issue.
This way I can also close the 1.0.0 project.
There is an operational issue for this alread: https://github.com/threefoldtech/itenv_tfgridv2/issues/29
If you want to track it in this project as well, please create a seperate issue.

**Comment by sabrinasadik** [comment]
Can this be closed then now?
